tests/test_deserialize_sequence.cc

```diff
@@ -27,4 +27,12 @@
 #include "deserialize_sequence.h"
 
 using namespace task;
-using namespace task::literals;
+
+TEST_CASE("parse_tags()", "[deserialize_sequence]")
+{
+    REQUIRE(parse_tags("").empty());
+    REQUIRE(parse_tags(" tag1 tag2\ttag3\n")
+        == std::vector{ Tag{ "tag1" }, Tag{ "tag2" }, Tag{ "tag3" } });
+    REQUIRE(parse_tags(" ta*g1 tag2\ttag3\nYet-Another-Tag")
+        == std::vector{ Tag{ "tag2" }, Tag{ "tag3" }, Tag{ "yet-another-tag" } });
```

Comment:
Idea for test: have an invalid tag

Reply:
Line 36 already has an invalid tag, and it is silently not inserted into the vector.

Reply:
Has been rewritten to some degree because the behavior has changed (parse_tags throws now).